.evergreen/find-python3.sh

```diff
@@ -81,15 +81,30 @@ is_venv_capable() (
   local -r tmp="$(mktemp -d)"
   trap 'rm -rf "$tmp"' EXIT
 
-  "$bin" -m venv "$tmp" || return
+  if [[ "$OSTYPE" == cygwin ]]; then
+    local -r real_path="$(cygpath -aw "$tmp")" || return
+    "$bin" -m venv "$real_path" || return
+  else
+    "$bin" -m venv "$tmp" || return
+  fi
+
+  # Sanity check: on some environments (such as Cygwin) creation of the virtual
+  # environment may succeed but place the environment in an unexpected location.
+  if [[ -n "$(find "$tmp" -maxdepth 0 -type d -empty 2>/dev/null)" ]]; then
```

Comment:
Can you show an example of this happening? Regardless can we remove this check because it's already handled by the if/elif/else below?

Reply:
Yeah I would prefer removing it because it simplifies the script and we don't do anything special for an empty dir.